sys-boot/tboot/tboot-1.10.3.ebuild

```
@@ -52,15 +49,15 @@ src_install() {
 	emake DISTDIR="${D}" install
 
 	dodoc "${DOCS[@]}"
-	dodoc docs/*.txt lcptools/*.pdf
+	dodoc docs/*.{txt,md}
 
-	cd "${ED}"
+	cd "${D}"
```

```suggestion
	cd "${ED}" || die
```
why did you change back to `D`?

---

Wasn't intentional, not sure what happened. I changed it back